acceptance/acceptance_test.go

```diff
@@ -368,33 +374,28 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	err = CopyDir(dir, tmpDir, inputs, outputs)
 	require.NoError(t, err)
 
+	cfg, user := internal.PrepareServerAndClient(t, config, LogRequests, tmpDir)
+	testdiff.PrepareReplacementsUser(t, &repls, user)
+	testdiff.PrepareReplacementsWorkspaceConfig(t, &repls, cfg)
+
+	// In inprocess mode, we need to modify the environment of the test runner, so that the script sees the
```

Comment:
That's not correct, even in inprocess mode the script is launched as a subprocess (it calls back into main process though) and the script passes all of the environments to the callback, so you don't need to set it up specifically for inprocessMode.

Reply:
Addressed in https://github.com/databricks/cli/pull/2822